pkg/controller/pxc/controller.go

```diff
@@ -844,10 +846,16 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileConfigMap(cr *api.PerconaXtraDB
 		if err != nil {
 			return errors.Wrap(err, "pxc config map")
 		}
+	} else {
+		if err := deleteConfigMapIfExists(r.client, cr, pxcConfigName); err != nil {
+			return errors.Wrap(err, "delete pxc config map")
+		}
 	}
 
+	var proxysqlConfigName = ls["app.kubernetes.io/instance"] + "-proxysql"
```

Comment:
please, do not use `var`, use `:=` operator

Reply:
36ed173